pyproject.toml

```diff
@@ -166,6 +166,7 @@ version = { attr = "invokeai.version.__version__" }
 
 [tool.setuptools.package-data]
 "invokeai.app.assets" = ["**/*.png"]
+"invokeai.app.services.workflow_records.default_workflows" = ["*.json"]
```

Comment:
This is needed to bundle the default workflows. I tested this by building the app as a wheel and confirming the workflows are synced correctly.